scalalib/src/mill/scalalib/publish/settings.scala

```diff
@@ -58,11 +58,25 @@ case class License(
     distribution: String = "repo"
 )
 
-case class SCM(
+object SCM {
+  @deprecated("use VersionControl", "0.1.0")
```

Comment:
Not sure we should deprecate this

Reply:
I prefer VersionControl, it's self-explanatory.